addons/mrp/models/mrp_production.py

```diff
@@ -1057,3 +1079,8 @@ def _render_note_exception_quantity_mo(rendering_context):
             return self.env.ref('mrp.exception_on_mo').render(values=values)
 
         self.env['stock.picking']._log_activity(_render_note_exception_quantity_mo, documents)
+
+    @api.model
+    def _search_delay_alert_date(self, operator, value):
+        late_stock_moves = self.env['stock.move'].search([('delay_alert_date', '!=', False)])
+        return ['|', ('move_raw_ids', 'in', late_stock_moves.ids), ('move_finished_ids', 'in', late_stock_moves.ids)]
```

Comment:
if you don't use operator and value here when using this field in a custom filter things will be weird: always the same result whatever the filter's value